bash/atomic

```diff
@@ -94,8 +94,13 @@ __atomic_image_repos_and_tags() {
 }
 
 __atomic_image_repos_and_tags_and_ids() {
-	local images="$(__docker_q images -a --no-trunc | awk 'NR>1 { print $3; if ($1 != "<none>") { print $1; print $1":"$2 } }')"
-	COMPREPLY=( $(compgen -W "$images" -- "$cur") )
+	local images
+	if test $_system == "0"; then
+		images="$(__docker_q images -a --no-trunc | awk 'NR>1 { print $3; if ($1 != "<none>") { print $1; print $1":"$2 } }')"
+	else
+		images="$(ls -1 /var/lib/containers/atomic/ | grep -v \.[01]$)"
+	fi
```

Comment:
This path should come from the config, although I would not hold up the merge to get it.